maximbircu/devtools-library
# Set SwiftLint paths

Closes: https://github.com/maximbircu/devtools-library/issues/85


### What has been done
 1. Added `.swiftlint.yml` files in `devtools/ios`, `samples/ios` where is specified the paths to lint.
 2. Fixed the Swift lint warnings.

### How to test
1.  Open `devtools/ios`. Run `Pods/SwiftLint/swiftlint`. No warnings should appear.
2. Open `samples/ios`. Run `Pods/SwiftLint/swiftlint`. No warnings should appear.
